quupe-RN/js/components/UpdateProfileForm/UpdateProfileForm.js

```diff
@@ -0,0 +1,192 @@
+import React, { Component } from 'react';
```

Comment:
your component name is BioForm and your file name is UpdateProfileForm.js. Please change your component name and make the respective changes to your project.